### ISSUE TITLE
Is it possible to get verbose scan output to shell?

### ISSUE THREAD

**Issue by 13CSherman** [dependency-check/DependencyCheck#5561]
I'm not too sure if this has been asked before - it probably has been. But I=I was wondering if there is a way to output verbose information onto shell while the scanner is running without using --log flag.

**Comment by aikebah** [comment]
No, only the --log is available for verbose output of the CLI (to a file)
